helm/prometheus-rules/templates/alerting-rules/grafana.management-cluster.rules.yml

```
@@ -24,7 +24,7 @@ spec:
         cancel_if_cluster_status_updating: "true"
         cancel_if_scrape_timeout: "true"
         cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
```

Can we also please change this, no need to wake up anyone when grafana is down.

```suggestion
        cancel_if_outside_working_hours: "true"
```